integrationTests/chainSimulator/esdt/bridge_test.go

```diff
@@ -1,190 +0,0 @@
-package esdt
```

Comment:
Why did you delete it to move it in another package `bridge`?
Please do a `git checkout feat/esdt-prefix integrationTests/chainSimulator/esdt/bridge_test.go` and revert it.
These are esdt tests for prefixed tokens(the fact that we use bridge contracts it's not in the scope, that's just the tool we used for testing the  `feat/esdt-prefix` ), Should've left it here, along with other tests